charts/rancher-webhook/Chart.yaml

```diff
@@ -11,4 +11,4 @@ annotations:
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/rancher-version: ">= 2.9.0-0 < 2.10.0-0"
-  catalog.cattle.io/kube-version: ">= 1.23.0-0 < 1.29.0-0"
```

Comment:
Imo this should be done as part of https://github.com/rancher/webhook/pull/366. The current version of webhook doesn't support 1.29 so I wouldn't expect the chart to say it does.